### PR TITLE
Changed link to best practices talk to YouTube link

### PR DESCRIPTION
**Pull request by svametcalf**
Because the google events page throws a 404, I have updated the docs with a direct link to the video on YouTube

---
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/216)

### CODE REVIEW

**Comment by svametcalf** [comment]
I signed it!

**Comment by svametcalf** [comment]
I have added the link to the slides. 

NOTE: Github was acting funny about a change that was made in the web editor, so I had to revert and revert the revert in order to force github to recognize the changes. These extra commits can be squashed without any issue.

File: src/main/markdown/doc/latest/DevGuideTesting.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ separation of concerns, dependency injection and the like will benefit
 your GWT app just as they would any other, perhaps even more than usual.
 
 For some tips along these lines take a look at
-the [Best
-Practices For Architecting Your GWT App](http://code.google.com/events/io/2009/sessions/GoogleWebToolkitBestPractices.html) talk given at Google I/O
-in May of 2009. And keep an eye on this site for more more articles in
+the *Best Practices For Architecting Your GWT App* 
+([video](https://www.youtube.com/watch?v=PDuhR18-EdM) or 
+[slides](http://dl.google.com/io/2009/pres/Th_0200_GoogleWebToolkitArchitecture-BestPracticesForArchitectingYourGWTApp.pdf)) given at Google I/O in May of 2009. And keep an eye on this site for more more articles in
```
Comment (tbroyer) [inline src/main/markdown/doc/latest/DevGuideTesting.md]: It looks like you removed the word "talk"?

Comment (svametcalf) [inline src/main/markdown/doc/latest/DevGuideTesting.md]: It looks like I did. I will update.

**Comment by manolo** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/gwtproject/gwt-site/216#-:-Ke35oNMbZEkE1Zon6Fg:bnfp4nl)*

**Pull request merged**